### ISSUE TITLE
conda migration

### ISSUE THREAD

**Issue by pkerins** [wri/UrbanLandUse#4]
We are now using a conda environment to manage our Python packages, whereas before we were installing things with pip. As a result, some imports no longer work as expected (eg `import keras`) and need to be replaced. Additionally, code referencing the original imports may also no longer be able to execute (eg `network = K.load_model(network_filename, custom_objects={'loss': 'categorical-crossentropy'})`). Both pieces need to be corrected, whether in util files or notebooks.
